pkg/controllers/user/helm/common/common.go

```diff
@@ -249,3 +291,43 @@ func escapeCommas(value string) string {
 	}
 	return strings.Replace(value, ",", "\\,", -1)
 }
+
+func createKustomizeFiles(tempDirs *HelmPath, labelValue string) error {
+	var resources []string
+	var kustomizeSh []byte
+	resources = append(resources, "all.yaml")
+	k := Kustomization{Label{labelValue}, resources}
+	y, err := yaml.Marshal(k)
+	if err != nil {
+		return err
+	}
+	kpath := filepath.Join(tempDirs.FullPath, "/kustomization.yaml")
+	err = ioutil.WriteFile(kpath, y, 0644)
+	if err != nil {
+		return err
+	}
+	logrus.Debugf("successfully created kustomization.yaml in %s", kpath)
+
+	if os.Getenv("CATTLE_DEV_MODE") != "" {
+		kustomizeSh, err = ioutil.ReadFile("./package/kustomize.sh")
+		if err != nil {
+			kustomizeSh, err = ioutil.ReadFile(filepath.Join(os.Getenv("DAPPER_SOURCE"), "package/kustomize.sh"))
+			if err != nil {
+				return err
+			}
+		}
+	} else {
+		kustomizeSh, err = ioutil.ReadFile("/usr/bin/kustomize.sh")
+		if err != nil {
+			return err
+		}
+	}
+	tmpString := strings.ReplaceAll(string(kustomizeSh), " . ", fmt.Sprintf(" %s ", tempDirs.InJailPath))
+	tmpString = strings.ReplaceAll(tmpString, "all.yaml", filepath.Join(tempDirs.InJailPath, "all.yaml"))
+	err = ioutil.WriteFile(filepath.Join(tempDirs.FullPath, "/kustomize.sh"), []byte(tmpString), 0755)
```

Comment:
Is this still needed with fixing up the command call? The script should just be good to go and not require reading and modifying. If anything you can pass a path to the script if required. The file is also linked into this dir so if you modify it with something other than `/` it could be bad times for others.